tests/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+---
+version: '3'
+name: "minio_tests"
+services:
+  minio:
+    image: minio/minio
+    ports:
+      - "9000:9000"
+      - "9001:9001"
+    environment:
+      MINIO_ROOT_PASSWORD: Strong#Pass#2022
+      MINIO_ROOT_USER: masoud
+      MINIO_CI_CD: true
+      MINIO_NOTIFY_WEBHOOK_ENABLE_miniorustest: on
+      MINIO_NOTIFY_WEBHOOK_ENDPOINT_miniorustest: http://example.org/
+
+    volumes:
+      - ./private.key:/certs/private.key
+      - ./public.crt:/certs/public.crt
+    command: server /data/test-xl/{1...4} --certs-dir /certs
```

Comment:
We do not need anything like this.

Reply:
https://github.com/minio/minio-rs/pull/32#issuecomment-1578311770